.github/workflows/integration.yaml

```diff
@@ -114,6 +114,123 @@ jobs:
               test -p ibc-integration-test --no-fail-fast -- \
               --nocapture --test-threads=2
 
+  multi-chains-test:
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/master'
```

Comment:
Can we comment this to test on this branch, and then uncomment it again before merging once this pass?